src/app/_components/CardSpecialist/SocialsList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+import { cn } from '@utils/cn';
+import { specialistSocialsPropType } from '@/app/_components/CardSpecialist/prop-types';
+
+export function SocialsList({ socials, className }) {
+  const socialsFiltered = socials.filter(social => !!social.href);
+
+  return (
+    <ul
+      className={cn('sm:hidden md:flex md:flex-row md:items-center md:justify-center md:gap-[5px] lg:gap-3', className)}
+    >
+      {socialsFiltered?.map((item, idx) => (
```

Comment:
?. is not required, if socials is not an array it would failed on the line 8

Reply:
Fixed